scout/server/links.py

```diff
@@ -447,6 +479,32 @@ def ucsc_link(variant_obj, build=None):
     return url_template.format(this=variant_obj)
 
 
+def cbioportal(hgnc_symbol, protein_sequence_name):
```

Comment:
This one could be tested

Reply:
Indeed! Will do!

Reply:
I settled with testing the function it calls. The rest should be very straightforward (famous last words).